tools/cldr-apps/src/main/java/org/unicode/cldr/web/XPathTable.java

```diff
@@ -140,7 +140,6 @@ private void setupDB(Connection ourConn) throws SQLException {
             sql = null;
             s.close();
             s = null;
-            ourConn.commit();
```

Comment:
when called from SurveyMain, ourConn = getAConnection

but when called from TestXPathTable (twice) and TestSTFactory, ourConn = getDBConnection

seems like a bug either way; the callers need to be consistent?

Reply:
fixed